### ISSUE TITLE
Routing does not work with arbitrary servlet context path

### ISSUE THREAD

**Issue by elizarov** [ktorio/ktor#322]
Take https://github.com/ktorio/ktor-samples/tree/master/deployment/jetty-war sample project and modify context path for its deployments in `build.gradle` file:

```
gretty {
    contextPath = '/other'
}
```

Run run the sample with `./gradlew :jetty-war:run` and try to open http://localhost:8080

Result: Not found

Expected: Opens an example page that is configured in code in the following way:

```
install(Routing) {
        get("/") { ... }
}
```

Routining shall be relative to the context path when running under application server like Jetty.

**Comment by cy6erGn0m** [comment]
Will be fixed as per da1888df7eab6324febdfe0e9ed97cc686a26963 and f5cd6014e67814fcfe478b68f054ad0bac56b653

**Comment by cy6erGn0m** [comment]
Fixed in 1.2.0-rc2
